docs/src/pages/internals/mapping.js

```diff
@@ -393,7 +399,7 @@ deserializer.Deserialize(bytes); // throws OverflowException`}</Highlight>
     </ul>
 
     <h2 id='records'>Records</h2>
-    <p>Chr.Avro maps .NET classes and structs to Avro’s <Highlight inline language='avro'>"record"</Highlight> type by matching each record field to a field or property on the type. The rules:</p>
+    <p>Chr.Avro maps .NET classes and structs to Avro’s <Highlight inline language='avro'>"record"</Highlight> type by attempting to find a constructor that has parameter names matching all record fields (extra optional parameters allowed). If no matching constructors are found then it will attempt to use the parameterless constructor and match each record field to a field or property on the type. The rules:</p>
```

Comment:
update rest of section too: member names -> parameter or member names; members -> parameters or members, etc.